redbox-core/redbox/app.py

```diff
@@ -42,7 +79,7 @@ async def run(
         activity_event_callback=_default_callback,
     ) -> RedboxState:
         final_state = None
-        async for event in self.graph.astream_events(input, version="v2"):
+        async for event in self.graph.astream_events(input=input, version="v2", config={"recursion_limit": 50}):
```

Comment:
just thinking out loud, no need to do anything: 50 seems pretty huge? (in a separate PR lets move this into the AISettings)

Reply:
> each step

ahhh, got it, thx